src/Core/Prefab/Planar/PlanarTileBuilder.js

```diff
@@ -1,73 +1,72 @@
+/* eslint-disable class-methods-use-this */
 import * as THREE from 'three';
 import OBB from 'Renderer/OBB';
 import Coordinates from 'Core/Geographic/Coordinates';
 import Extent from 'Core/Geographic/Extent';
 
-function PlanarTileBuilder() {
-    this.tmp = {
-        coords: new Coordinates('EPSG:4326', 0, 0),
-        position: new THREE.Vector3(),
-        normal: new THREE.Vector3(0, 0, 1),
-    };
-
-    this.type = 'p';
-}
-
-PlanarTileBuilder.prototype.constructor = PlanarTileBuilder;
-
-// prepare params
-// init projected object -> params.projected
-PlanarTileBuilder.prototype.Prepare = function Prepare(params) {
-    params.nbRow = 2 ** (params.zoom + 1.0);
-    params.projected = new THREE.Vector3();
-};
+const quaternion = new THREE.Quaternion();
+const center = new THREE.Vector3();
 
+class PlanarTileBuilder {
+    constructor() {
+        this.type = 'p';
+        this.tmp = {
+            coords: new Coordinates('EPSG:4326', 0, 0),
+            position: new THREE.Vector3(),
+            normal: new THREE.Vector3(0, 0, 1),
+        };
+    }
+    // prepare params
+    // init projected object -> params.projected
+    Prepare(params) {
```

Comment:
Can you take this opportunity to also rename methods of this class to lowercase as first letter ?

Reply:
ok